_rules/disable-shortcut-670a30.md

```diff
@@ -0,0 +1,326 @@
+---
+id: 670a30
+name: Printable keys only shortcut can be disabled
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, a mechanism to disable the shortcut exists.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys.
+
+## Expectation
+
+For the test target if any [keyboard shortcut][] requires pressing only [printable character][] keys, a [mechanism][] to disable the shortcut exists.
```

Comment:
I think expectation should mimic the formulation of the expectation in 1e9941:
"For each UI component that is descendant of the test target, if a printable key shortcut exists, it can be disabled" or something like that.

Same goes for other rules. Because they are pretty similar, a similar expectation will help…